src/cargo/util/config/de.rs

```diff
@@ -195,11 +195,35 @@ impl<'de, 'config> de::Deserializer<'de> for Deserializer<'config> {
         }
     }
 
+    fn deserialize_enum<V>(
+        self,
+        name: &'static str,
+        _variants: &'static [&'static str],
+        visitor: V,
+    ) -> Result<V::Value, Self::Error>
+    where
+        V: de::Visitor<'de>,
+    {
+        if name == "Strip" {
+            let cv = self.config.get_cv(&self.key)?;
+            if let Some(cv) = cv {
+                match cv {
+                    CV::String(s, ..) => visitor.visit_enum(s.into_deserializer()),
+                    _ => todo!("Config value is of wrong type"),
+                }
+            } else {
+                todo!("Config value not found")
```

Comment:
This can raise `ConfigError::missing`.  See the `deserialize_method` macro for an example of how strings and integers are implemented.  I think the implementation here will look almost exactly the same as that macro, except it should call `visit_enum(s.into_deserializer())`.